repos/fdbt-site/package.json

```diff
@@ -57,28 +57,27 @@
         "govuk-frontend": "^3.13.0",
         "helmet": "^3.22.0",
         "jsdom-global": "^3.0.2",
-        "jsonwebtoken": "^8.5.1",
-        "jwks-rsa": "^1.12.1",
+        "jsonwebtoken": "^9.0.2",
+        "jwks-rsa": "^3.0.1",
         "lodash": "^4.17.21",
         "moment": "^2.28.0",
         "mysql2": "^2.2.5",
-        "next": "^11.1.2",
-        "next-compose-plugins": "^2.2.0",
+        "next": "^12.1.2",
         "next-images": "^1.3.1",
         "nocache": "^2.1.0",
         "nodemailer": "^6.4.16",
         "nookies": "^2.3.0",
         "papaparse": "^5.3.0",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
-        "react-portal": "^4.2.1",
-        "react-test-renderer": "^17.0.2",
+        "react-portal": "4.2.1",
```

Comment:
Can we get rid of this now or is it being used elsewhere?

Reply:
done